<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServicesDialog</class>
 <widget class="QDialog" name="ServicesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="servicesLabel">
       <property name="text">
        <string>Список услуг</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeButton">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="servicesLabel_2">
         <property name="text">
          <string>Услуга</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="serviceLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="servicesLabel_3">
         <property name="text">
          <string>Тарификация</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="meterRadioButton">
         <property name="text">
          <string>счётчик</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="tariffRadioButton">
         <property name="text">
          <string>тариф</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="servicesLabel_5">
         <property name="text">
          <string>Ед. измерения</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="unitComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="servicesLabel_4">
         <property name="text">
          <string>Стоимость за ед.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="priceLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="tariffSelectButton">
         <property name="text">
          <string>выбор тарифа</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ServicesDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
